base/linalg/bidiag.jl

```diff
@@ -630,14 +630,14 @@ SpecialArrays = Union{Diagonal,
 end
 
 # for historical reasons:
```

Comment:
unrelated, but we should remember to check how long this has been around and see about deprecating it later